src/engine/SCons/Tool/gnulink.py

```diff
@@ -57,21 +55,22 @@ def generate(env):
 
     # OpenBSD doesn't usually use SONAME for libraries
     use_soname = not sys.platform.startswith('openbsd')
-    link._setup_versioned_lib_variables(env, tool = 'gnulink', use_soname = use_soname)
+    link._setup_versioned_lib_variables(env, tool='gnulink', use_soname=use_soname)
     env['LINKCALLBACKS'] = link._versioned_lib_callbacks()
 
-    # For backward-compatibility with older SCons versions
-    env['SHLIBVERSIONFLAGS'] = SCons.Util.CLVar('-Wl,-Bsymbolic')
-    
+    # # For backward-compatibility with older SCons versions
```

Comment:
I wonder what the back-compat comment was about - the value, or setting SHLIBVERSIONFLAGS at all?

Reply:
There were a few iterations of the versioned shared library logic. I think the second iteration removed that flag, and then we added it back because someone requested it?  And/or a test depended on it after iteration 2 was tested.. Something like that.